ci/requirements-ac-test.txt

```diff
@@ -91,7 +93,7 @@ rawpy==0.16.0
     # via -r tools/accuracy_checker/requirements.in
 regex==2021.3.17
     # via nltk
-scikit-image==0.17.2
+scikit-image==0.17.2 ; python_version == "3.6"
```

Comment:
Where does this come from? Accuracy Checker's requirements don't have this environment marker.

Reply:
I generated these files on package with new requirements for all components. This came from PR to Accuracy Checker (#2304): https://github.com/openvinotoolkit/open_model_zoo/pull/2304/files#diff-94b0d1f970ed85cbabda97cbe06b76d9b687806c12af077538ca2f8f311d6e2dR5-R6

Reply:
That makes the PR rather confusing. Please only keep the changes to those `ci/requirements-*` files that are directly affected by the changes to `tools/downloader/*` and move the other `ci/requirements-*` changes to the relevant PRs.

Reply:
Done